frontend/src/components/EditableField.js

```diff
@@ -169,15 +169,17 @@ export const EditableField = withStyles(efStyles)(EditableFieldBase);
 export class WorksheetEditableField extends React.Component<{
     uuid: string,
     fieldName: string,
+    dataType: string,
 }> {
     buildPayload(value) {
+        console.log(this.props.dataType);
         return {
             data: [
                 {
                     id: this.props.uuid,
                     type: 'worksheets',
                     attributes: {
-                        [this.props.fieldName]: value,
+                        [this.props.fieldName]: serializeFormat(value, this.props.dataType),
```

Comment:
What is the serializeFormat trying to do here, it seems the value is still a string?

Reply:
The value is a string, but the dataType that we want to interpret the value is not.

Reply:
Oh, so value is not a string, but we want to serialize it into a string?

Reply:
Yes - It's a little confusing but this is what we did for `BundleEditableField`